README.md

```diff
@@ -35,14 +35,26 @@ instructions](docs/installation.md).
 
```

Comment:
Isn't this section supposed to be about listing and documenting the individual configuration options rather than the simplest possible steps to the application up and running (which is what's described in the installation guide)?